lib/services/address/index.js

```diff
@@ -50,7 +51,8 @@ AddressService.dependencies = [
 
 AddressService.PREFIXES = {
   OUTPUTS: new Buffer('02', 'hex'),
-  SPENTS: new Buffer('03', 'hex')
+  SPENTS: new Buffer('03', 'hex'),
```

Comment:
We might want to include comments as to what the key and value is for each of these indexes. It can get kind of confusing.